website/docs/d/pi_disaster_recovery_locations.html.markdown

```diff
@@ -39,6 +39,6 @@ In addition to all argument reference list, you can access the following attribu
   - `location` - (String) The region zone of a site.
   - `replication_sites` - List of Replication Sites.
   
-    Nested scheme for `replication_sites`:
-    - `is_active` - (Boolean) Indicates the location is active or not, `true` if location is active, otherwise it is `false`.
-    - `location` - (String) The region zone of the location.
\ No newline at end of file
+        Nested scheme for `replication_sites`:
+        - `is_active` - (Bool) Indicates the location is active or not, `true` if location is active, otherwise it is `false`.
```

Comment:
I think we might have to keep it `Boolean`
```suggestion
        - `is_active` - (Boolean) Indicates the location is active or not, `true` if location is active, otherwise it is `false`.
```